plugins/BEdita/Core/src/Command/ObjectsDeleteCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+declare(strict_types=1);
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2024 Atlas Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\Core\Command;
+
+use Cake\Command\Command;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Console\ConsoleOptionParser;
+use Cake\Database\Expression\QueryExpression;
+use Cake\I18n\FrozenDate;
+use Cake\ORM\Locator\LocatorAwareTrait;
+
+/**
+ * ObjectsDelete command.
+ */
+class ObjectsDeleteCommand extends Command
+{
+    use LocatorAwareTrait;
+
+    /**
+     * @inheritDoc
+     */
+    public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionParser
+    {
+        return $parser
+            ->addOption('since', [
+                'help' => 'Delete objects in trash since this date',
+                'required' => false,
+            ])
+            ->addOption('type', [
+                'help' => 'Delete objects in trash by type',
+                'required' => false,
+            ]);
```

Comment:
It should be possible to set the default in the option parser itself ([docs](https://book.cakephp.org/4/en/console-commands/option-parsers.html#using-options)). Additionally, it would be cool to have `--type` flag be allowed multiple times, and as such treated as an array in the code:

```suggestion
                'required' => false,
                'default' => '-1 month',
            ])
            ->addOption('type', [
                'help' => 'Delete objects in trash by type',
                'required' => false,
                'multiple' => true,
            ]);
```